### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by weissjeffm**
A couple of readme fixes for issues I ran into:

* bitcoind node with pruning enabled doesn't work properly

* Number of confirmations needed for a channel to enter NORMAL state is now 3 but readme still says 1.

### CODE REVIEW

**Comment by cdecker** [comment]
The "Starting `lightningd`" section contains a hint that pruning isn't supported, but I'm happy to double that further up.

ACK 821c164

**Comment by weissjeffm** [comment]
Ah yes didn't notice that it was mentioned below. I agree it's probably good to mention it as early as possible, since anyone following the readme from scratch might sync the entire chain pruned before realizing that they cannot continue that way.

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ cli/lightning-cli fundchannel <node_id> <amount>
 
 This opens a connection and, on top of that connection, then opens a channel.
 The funding transaction needs 1 confirmations in order for the channel to be usable, and 6 to be broadcast for others to use.
-You can check the status of the channel using `cli/lightning-cli listpeers`, which after 1 confirmation should say that `state` is `CHANNELD_NORMAL`; after 6 confirmations you can use `cli/lightning-cli listchannels` to verify that the `public` field is now `true`.
+You can check the status of the channel using `cli/lightning-cli listpeers`, which after 3 confirmations should say that `state` is `CHANNELD_NORMAL`; after 6 confirmations you can use `cli/lightning-cli listchannels` to verify that the `public` field is now `true`.
```
Comment (rustyrussell) [inline README.md]: Maybe "3 confirmations (1 on testnet)"?

**Comment by practicalswift** [comment]
ACK 821c1643746c58f2907e49bdeb349765fdeebd30 modulo @rustyrussell's feedback regarding number of confirmations on testnet

**Comment by cdecker** [comment]
@weissjeffm do you want to add the proposed addition or shall I merge as is?

**Comment by weissjeffm** [comment]
Now says 3 for mainnet, 1 for testnet.

**Comment by cdecker** [comment]
ACK 5be49de

**Pull request merged**